source/_integrations/zwave_js.markdown

```diff
@@ -44,18 +44,56 @@ ha_integration_type: hub
+
+Follow these steps:
+
+1. Open the Home Assistant user interface.
+1. Plug the Z-Wave dongle into the device running Home Assistant.
+   * Most likely, your dongle will be recognized automatically. On the user interface, you will be asked if you want to set up this device with the Z-Wave JS add-on. Select **Submit**.
+   * If your dongle is not recognized, follow these steps:
+
+{% details "Manual setup steps" %}
+Use this My button:
+
+{% my config_flow_start badge domain="zwave_js" %}, or follow these steps:
+
+* Browse to your Home Assistant instance.
+* In the sidebar click on _**{% my config icon %}**_.
+* From the configuration menu select: _**{% my integrations %}**_.
+* In the bottom right, click on the
+  _**{% my config_flow_start icon domain="zwave_js" %}**_ button.
+* From the list, search and select _**"Z-Wave"**_.
+* Follow the instructions on screen to complete the set up.
+
+{% enddetails %}
+
+3. Wait for the installation to complete.
+1. You are prompted for network security keys.
+   * If you are using Z-Wave for the first time, leave all the fields empty and select **Submit**. The system will generate network security keys for you.
+   * If this Z-Wave dongle has already been paired with secure devices, you need to enter the previously used network key as the S0 network key. S2 security keys will be automatically generated for you.
+   * Make sure that you keep a backup of these keys in a safe place in case you need to move your Z-Wave dongle to another device. Copy and paste them somewhere safe.
+1. Wait for the Z-Wave JS add-on to start up.
+1. Once the installation is complete, the **Device info** of the Z-Wave controller is shown.
+   * You successfully installed the Z-Wave integration and the Z-Wave JS add-on.
+   * You can now add devices to the Z-Wave network.
 
 <p class='note'>
-While your Z-Wave mesh is permanently stored on your stick, the additional metadata is not. When the Z-Wave integration starts up the first time, it will interview your entire Z-Wave network. Depending on the number of devices paired with the Z-Wave stick, this can take a while. You can speed up this process by manually waking up your battery-powered devices. Most of the time this is a press on the button on those devices (see their manual). It is not necessary to exclude/re-include devices from the mesh.
+While your Z-Wave mesh is permanently stored on your dongle, the additional metadata is not. When the Z-Wave integration starts up the first time, it will interview your entire Z-Wave network. Depending on the number of devices paired with the Z-Wave dongle, this can take a while. You can speed up this process by manually waking up your battery-powered devices. Most of the time, this is a press on the button on those devices (see their manual). It is not necessary to exclude and re-include devices from the mesh.
```

Comment:
```suggestion
While your Z-Wave mesh is permanently stored on your dongle, the additional metadata is not. When the Z-Wave integration starts up the first time, it will interview your entire Z-Wave network. Depending on the number of devices paired with the Z-Wave dongle, this can take a while. You can speed up this process by manually waking up your battery-powered devices. Most of the time, this is a button press on those devices (see their manual). It is not necessary to exclude and re-include devices from the mesh.
```